core/cloudflow-akka/src/main/resources/akka-cluster-k8.conf

```diff
@@ -1,20 +1,24 @@
 akka {
-  cluster {
-    min-nr-of-members = 1
+  actor {
+    provider = "cluster"
+    # FIXME remove when Avro serializer has been added, issue #527
+    allow-java-serialization = on
```

Comment:
It might be better to turn it off here and add it as on in the examples that use java serialization in the cluster because of avro (which we should not do and use protobuf instead)

Reply:
This is an interesting one. It does not matter whether we use Avro or protobuf for messaging. This is used for serialization of Scala classes for persistence. I introduced CBOR support, so if your message is derived from it, then a proper serialization will be used. Leaving Java serialization in place is just a catcher, if you have not derived. Its really harmless